server/src/main/java/org/apache/iotdb/db/mpp/execution/schedule/DriverScheduler.java

```diff
@@ -82,8 +90,7 @@ public static DriverScheduler getInstance() {
 
   private DriverScheduler() {
     this.readyQueue =
-        new L2PriorityQueue<>(
-            MAX_CAPACITY, new DriverTask.SchedulePriorityComparator(), new DriverTask());
+        new MultilevelPriorityQueue(LEVEL_TIME_MULTIPLIER, MAX_CAPACITY, new DriverTask());
```

Comment:
Maybe L2PriorityQueue can be removed?

Reply:
Yes, it could be removed. Perhaps after we finish the design of metrics of schedule module and finish our tests?

Reply:
OK